test/prototype_common_utils.py

```diff
@@ -313,6 +315,38 @@ def make_image_loaders(
 make_images = from_loaders(make_image_loaders)
 
 
+def make_image_loader_for_interpolation(size="random", *, color_space=features.ColorSpace.RGB, dtype=torch.uint8):
+    size = _parse_spatial_size(size)
+    num_channels = get_num_channels(color_space)
+
+    def fn(shape, dtype, device):
+        num_channels, height, width = shape
+
+        top = int((0.8 * height) // 2)
+        left = int((0.8 * width) // 2)
+        bottom = height - top
+        right = width - left
+
+        image_pil = PIL.Image.new("L", (width, height))
+        draw = PIL.ImageDraw.Draw(image_pil)
+        draw.ellipse(((left, top), (right, bottom)), fill=255)
```

Comment:
This draws a white ellipse onto black background with roughly 10% empty space on each side of the border. This way we get a non-trivial shape like a rectangle, but still have a large interior region. Assuming that the only differences happen at he border, using the MAE, i.e. `agg_method="mean"`, for comparison, leads to small tolerances and thus more safety.

Reply:
As discussed offline, instead of using this artificial image, we should use a real one. 2eba005893d8eeeb8f9cc2fd290e6e11cc6538d5 uses Grace Hopper.